neurons/miners/miner.py

```diff
@@ -164,11 +164,17 @@ async def challenge(self, synapse: protocol.Challenge ) -> protocol.Challenge:
         try:
             bt.logging.info(f"challenge recieved: {synapse}")
 
-            synapse.output = self.graph_search.solve_challenge(
-                in_total_amount=synapse.in_total_amount,
-                out_total_amount=synapse.out_total_amount,
-                tx_id_last_4_chars=synapse.tx_id_last_4_chars
-            )
+            if self.config.network == NETWORK_BITCOIN:
+                synapse.output = self.graph_search.solve_challenge(
+                    in_total_amount=synapse.in_total_amount,
+                    out_total_amount=synapse.out_total_amount,
+                    tx_id_last_4_chars=synapse.tx_id_last_4_chars
+                )
+            if self.config.network == NETWORK_ETHEREUM:
+                synapse.output = self.graph_search.solve_challenge(
+                    in_total_amount=synapse.checksum,
```

Comment:
is this correct? total amount is eq checksum?

Reply:
Oh yes it should be `checksum`
Good catch, 👍